<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newmessage</class>
 <widget class="QDialog" name="newmessage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Beluga</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>240</width>
     <height>270</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>238</width>
      <height>268</height>
     </rect>
    </property>
    <widget class="QToolButton" name="send">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>242</y>
       <width>40</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QToolButton" name="more">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>242</y>
       <width>40</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>More</string>
     </property>
    </widget>
    <widget class="QPushButton" name="quickMsg">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>242</y>
       <width>40</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="belugaapp.qrc">
       <normaloff>:/BelugaApp/Resources/images/quick_msg.png</normaloff>:/BelugaApp/Resources/images/quick_msg.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>24</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="signature">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>242</y>
       <width>40</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="belugaapp.qrc">
       <normaloff>:/BelugaApp/Resources/images/got.png</normaloff>:/BelugaApp/Resources/images/got.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>24</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="msgFace">
     <property name="geometry">
      <rect>
       <x>105</x>
       <y>242</y>
       <width>24</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="belugaapp.qrc">
       <normaloff>:/BelugaApp/Resources/faces/face.png</normaloff>:/BelugaApp/Resources/faces/face.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
    <widget class="QScrollArea" name="scroll">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>240</width>
       <height>240</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="innerScroll">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>238</width>
        <height>238</height>
       </rect>
      </property>
      <widget class="QWidget" name="toMsg" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>240</width>
         <height>40</height>
        </rect>
       </property>
       <widget class="QPushButton" name="chooseContact">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>5</y>
          <width>24</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="belugaapp.qrc">
          <normaloff>:/BelugaApp/Resources/gui/down_arrow.png</normaloff>:/BelugaApp/Resources/gui/down_arrow.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="contactlist">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>5</y>
          <width>160</width>
          <height>30</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="to">
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>5</y>
          <width>30</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>To:</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="content" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>40</y>
         <width>240</width>
         <height>200</height>
        </rect>
       </property>
       <widget class="QWidget" name="mmsEmail" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>240</width>
          <height>60</height>
         </rect>
        </property>
        <widget class="QLabel" name="subject">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>0</y>
           <width>54</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Subject:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="subjectContent">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>0</y>
           <width>160</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QGroupBox" name="photo_audio">
         <property name="geometry">
          <rect>
           <x>15</x>
           <y>25</y>
           <width>140</width>
           <height>34</height>
          </rect>
         </property>
         <property name="title">
          <string/>
         </property>
         <widget class="QPushButton" name="photo">
          <property name="geometry">
           <rect>
            <x>5</x>
            <y>1</y>
            <width>40</width>
            <height>32</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="belugaapp.qrc">
            <normaloff>:/BelugaApp/Resources/images/picture.png</normaloff>:/BelugaApp/Resources/images/picture.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>40</width>
            <height>32</height>
           </size>
          </property>
         </widget>
         <widget class="QPushButton" name="audio">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>1</y>
            <width>40</width>
            <height>32</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="belugaapp.qrc">
            <normaloff>:/BelugaApp/Resources/images/audio.png</normaloff>:/BelugaApp/Resources/images/audio.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>40</width>
            <height>32</height>
           </size>
          </property>
         </widget>
         <widget class="QPushButton" name="video">
          <property name="geometry">
           <rect>
            <x>95</x>
            <y>1</y>
            <width>40</width>
            <height>32</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="belugaapp.qrc">
            <normaloff>:/BelugaApp/Resources/images/video.png</normaloff>:/BelugaApp/Resources/images/video.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>40</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </widget>
        <widget class="QPushButton" name="file">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>25</y>
           <width>40</width>
           <height>32</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="belugaapp.qrc">
           <normaloff>:/BelugaApp/Resources/images/file.png</normaloff>:/BelugaApp/Resources/images/file.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </widget>
       <widget class="QTextEdit" name="msgContent">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>80</y>
          <width>230</width>
          <height>120</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="value">
        <property name="geometry">
         <rect>
          <x>55</x>
          <y>60</y>
          <width>40</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="count">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>40</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Count:</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <zorder>send</zorder>
    <zorder>more</zorder>
    <zorder>quickMsg</zorder>
    <zorder>signature</zorder>
    <zorder>msgFace</zorder>
    <zorder>toMsg</zorder>
    <zorder>scroll</zorder>
   </widget>
  </widget>
  <widget class="QWidget" name="toEmail" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>300</y>
     <width>240</width>
     <height>110</height>
    </rect>
   </property>
   <widget class="QPushButton" name="chooseTo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>5</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="belugaapp.qrc">
      <normaloff>:/BelugaApp/Resources/gui/down_arrow.png</normaloff>:/BelugaApp/Resources/gui/down_arrow.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="toList">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>5</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="to_2">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>To:</string>
    </property>
   </widget>
   <widget class="QLabel" name="cc">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>40</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Cc:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="ccList">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="chooseCc">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="belugaapp.qrc">
      <normaloff>:/BelugaApp/Resources/gui/down_arrow.png</normaloff>:/BelugaApp/Resources/gui/down_arrow.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="bcc">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>75</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Bcc:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="bccList">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>75</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="chooseBcc">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>75</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="belugaapp.qrc">
      <normaloff>:/BelugaApp/Resources/gui/down_arrow.png</normaloff>:/BelugaApp/Resources/gui/down_arrow.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="belugaapp.qrc"/>
 </resources>
 <connections/>
</ui>
